### PR TITLE
Creating new file named maths for mathematical operation

### PR DESCRIPTION
**Pull request by lrouhier**
The Goal of this PR is to create a new file name maths.py that will be used to list useful mathematical functions for image manipulation.

### CODE REVIEW

**Comment by charleygros** [comment]
I can't think of other function to put in this file at this point.
But this is a great initiative.. and I am sure that in the future we will found things to put in it, now that it exists.

File: ivadomed/maths.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        kernlen (int): size of kernel
+
+    Returns:
+        ndarray: a 2D array of size (kernlen,kernlen)
+    """
+
+    x = np.linspace(-1, 1, kernlen + 1)
+    kern1d = np.diff(scipy.stats.norm.cdf(x))
+    kern2d = np.outer(kern1d, kern1d)
+    return imed_transforms.rescale_values_array(kern2d / kern2d.sum())
```
Comment (charleygros) [inline ivadomed/maths.py:22]: `rescale_values_array` could be a good candidate for this file!!!

File: ivadomed/maths.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import nibabel as nib
```
Comment (charleygros) [inline ivadomed/maths.py:1]: Could you please add this file to the API / RST? thanks :)

Comment (lrouhier) [reply ivadomed/maths.py:1]: Will do

File: ivadomed/maths.py (hunk 1 above)
Comment (lrouhier) [reply ivadomed/maths.py:22]: Good idea !!

Comment (lrouhier) [reply ivadomed/maths.py]: done in 8aed599

File: ivadomed/maths.py (hunk 2 above)
Comment (lrouhier) [reply ivadomed/maths.py]: done in 064412a

Comment (charleygros) [reply ivadomed/maths.py]: All good to me now :-)

File: docs/source/api_ref.rst [hunk 3]
```diff
@@ -89,3 +89,9 @@ Utils API
 
 .. automodule:: ivadomed.utils
 
+
+Maths API
+++++++++++++++++++++
```
Comment (jcohenadad) [inline docs/source/api_ref.rst:94]: i think the convention is to use the same number of characters as the text
```suggestion
++++++++++++++++++++
```

Comment (lrouhier) [reply docs/source/api_ref.rst]: done in: 503aaa8

**Pull request merged**